crates/motsu/src/shims.rs

```diff
@@ -122,7 +122,7 @@ unsafe extern "C" fn storage_flush_cache(_: bool) {
 ///
 /// # Panics
 ///
-/// May panic if fails to parse `MSG_SENDER` as an address.
+/// * If fails to parse `MSG_SENDER` as an address.
```

Comment:
```suggestion
/// * If it fails to parse `MSG_SENDER` as an address.
```
Is this even a correct panic description? In code it panics with `msg_sender should be set`

Reply:
I'll just drop all panics descriptions at `shims.rs` most of them are obsolete, shims are not public anyways and we will drop them soon